src/components/Garden/Feed/RightPanel.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useMemo } from 'react'
```

Comment:
why we removed the useMemo implementation since has a dependency on all this `account, largeMode, mobileMode, tabletMode`, which specially the account one can create a rerender of the whole component?

Reply:
I don't remember why, but I will add it again.

Reply:
Updated!